src/views/Details/SwapDetails.vue

```diff
@@ -164,7 +164,7 @@ export default {
         fees.push({
           asset: getNativeAsset(this.item.to),
           fee: toFee,
-          unit: chains[toChain].fees.unit
+          unit: getFeeAsset(this.item.to) ? getFeeAsset(this.item.to) : chains[toChain].fees.unit
```

Comment:
Isn't it better to store `getFeeAsset()` result in some local variable instead of calling it twice?